diff --git a/Cargo.toml b/Cargo.toml
index 2621118..0e5f56f 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -26,7 +26,7 @@ csv = "=1.1.6"
 # Pinned to 1.7.3, because 1.8.0 would required rustc 1.60+, but MSRV is 1.56.1.
 regex = "=1.7.3"
 reqwest = { version = "0.11", features = ["blocking"] }
-rusqlite = "0.25.4"
+rusqlite = { version = "0.25.4", features = ["bundled"] }
 serde_json = "1.0"
 sha2 = "0.10"
 
